hdate/hdate_string.py

```diff
@@ -1,121 +1,124 @@
         res = dw_str.decode("utf-8") + " " + res
     if is_short:
         return res
     if omer > 0 and omer < 50:
         res += " " + hebrew_number(omer, hebrew=is_hebrew, short=is_short)
         res += " " + "בעומר".decode("utf-8") if is_hebrew else " in the Omer"
     if holiday:
-        res += " " + holidays[is_hebrew][is_short][holiday-1].decode("utf-8")
+        res += " " + HOLIDAYS[is_hebrew][is_short][holiday-1].decode("utf-8")
     return res
 
 
 def get_omer_string(omer):
+    """Return a string representing the count of the Omer."""
     tens = ["", "עשרה", "עשרים", "שלושים", "ארבעים"]
     ones = ["", "אחד", "שנים", "שלושה", "ארבעה", "חמשה",
             "ששה", "שבעה", "שמונה", "תשעה"]
     if omer < 1 or omer > 49:
         raise ValueError('Invalid Omer day: {}'.format(omer))
-    t = omer / 10
-    o = omer % 10
-    s = 'היום '
+    ten = omer / 10
+    one = omer % 10
+    omer_string = 'היום '
     if omer > 10 and omer < 20:
-        s += ones[o] + ' עשר '
+        omer_string += ones[one] + ' עשר '
     elif omer > 9:
-        s += tens[t] + ' '
-        if o:
-            s += 'ו'
+        omer_string += tens[ten] + ' '
+        if one:
+            omer_string += 'ו'
     if omer > 2:
-        s += ones[o]
+        omer_string += ones[one]
         if omer < 11:
-            s += ' ימים '
+            omer_string += ' ימים '
         else:
-            s += ' יום '
+            omer_string += ' יום '
     elif omer == 1:
-        s += 'יום אחד '
+        omer_string += 'יום אחד '
     elif omer == 2:
-        s += 'שני ימים '
+        omer_string += 'שני ימים '
     if omer > 6:
-        s += 'שהם '
-        w = omer / 7
-        d = omer % 7
-        if w > 2:
-            s += ones[w] + ' שבועות '
-        elif w == 1:
-            s += 'שבוע אחד '
-        elif w == 2:
-            s += 'שני שבועות '
-        if d:
-            s += 'ו'
-            if d > 2:
-                s += ones[d] + ' ימים '
-            elif d == 1:
-                s += 'יום אחד '
-            elif d == 2:
-                s += 'שני ימים '
-    s += 'לעומר'
-    return s
+        omer_string += 'שהם '
+        weeks = omer / 7
+        days = omer % 7
+        if weeks > 2:
+            omer_string += ones[weeks] + ' שבועות '
+        elif weeks == 1:
+            omer_string += 'שבוע אחד '
+        elif weeks == 2:
+            omer_string += 'שני שבועות '
+        if days:
+            omer_string += 'ו'
+            if days > 2:
+                omer_string += ones[days] + ' ימים '
+            elif days == 1:
+                omer_string += 'יום אחד '
+            elif days == 2:
+                omer_string += 'שני ימים '
+    omer_string += 'לעומר'
+    return omer_string
 
 
 def get_parashe(parasha, short=False, hebrew=True):
+    """Get the string representing the parasha."""
     is_hebrew = 1 if hebrew else 0
     is_short = 1 if short else 0
-    res = parashaot[is_hebrew]
+    res = PARASHAOT[is_hebrew][parasha]
```

Comment:
This was found by running pylint which complained that the variable was not in use